doc/api/crypto.md

```diff
@@ -2061,13 +2061,13 @@ Any `TypedArray` or `DataView` instance may be passed as `buffer`.
 
 ```js
 const a = new Uint32Array(10);
-console.log(crypto.randomFillSync(a).toString('hex'));
+console.log(Buffer.from(crypto.randomFillSync(a).buffer).toString('hex'));
```

Comment:
Just using `.buffer` works in these cases, but I’m not sure we should include it in our documentation like this – most of the time when you’re doing this, you’ll want to consider `.byteLength` and `.byteOffset` as well?

Reply:
I just modified the samples to work without thinking to much about it. Printing a `Float64Array` formated as 'hex' is usually not a typical use. I will update the samples to use also `.byteLength` and `.byteOffset` but I'm not sure if doc of `randomFillSync()` is the right place to show how to work with typed arrays.